gui/src/routes.ts

```diff
@@ -22,6 +23,7 @@ export default {
   '/new-project': NewProject,
   '/new-project/:starter': NewProjectConfig,
   '/workspaces/:workspace': Workspace,
+  '/workspaces/:workspace/info': WorkspaceInfo,
```

Comment:
_nit:_ I think I prefer `EntityDetails` and `:entity/details` over `EntityInfo` and `:entity/info` - can we establish some nice naming convention for single-entity detail pages?